package.json

```diff
@@ -58,12 +58,12 @@
     "react-intl": "^6.1.1",
     "react-redux": "^7.2.8",
     "react-router-dom": "^5.3.3",
-    "redux": "^4.1.2",
+    "redux": "^4.2.0",
     "redux-thunk": "^2.4.1",
     "typesafe-actions": "^5.1.0",
     "unleash-proxy-client": "^2.2.0",
-    "victory-core": "^36.2.1",
-    "xstate": "^4.33.4"
+    "victory-core": "^36.6.7",
```

Comment:
This should match the version used by PatternFly.

I have a PR to update the version of Victory used by PatternFly, so let's wait until the next milestone to update this